
public class ExceptionDemo {

	public static void main(String[] args) {
		// TODO Auto-generated method stub

	try {
		int a=0;
		int b=2;
		int c=a/b;
		System.out.println(c);
	}
catch(ArithmeticException e) {
		
	
	}System.out.println("It is  my pen");
	}
}
